xbmc/guilib/GUIFontCache.cpp

```diff
@@ -154,7 +171,11 @@ Value &CGUIFontCacheImpl<Position, Value>::Lookup(Position &pos,
     // Cache miss
     dirtyCache = true;
     CGUIFontCacheEntry<Position, Value> *entry = nullptr;
-    if (!m_list.ageMap.empty() && (nowMillis - m_list.ageMap.begin()->first) > FONT_CACHE_TIME_LIMIT)
+
+    const auto duration =
+        std::chrono::duration_cast<std::chrono::milliseconds>(now - m_list.ageMap.begin()->first);
```

Comment:
you can't execute `m_list.ageMap.begin()->` if `m_list.ageMap` is empty

Reply:
Thanks! fixed.